### PR TITLE
feat: New Icons

### PR DESCRIPTION
**Pull request by kof**
## Description

- [x] left sidebar tab icons

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 0000)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: apps/builder/app/builder/sidebar-left/help-popover.tsx [hunk 1]
```diff
@@ -45,29 +45,33 @@ export const HelpPopover = ({
             formAction="https://www.youtube.com/playlist"
             name="list"
             value="PL4vVqpngzeT4sDlanyPe99dYl8BgUYCac"
-            prefix={<Youtube1cIcon />}
+            prefix={<Youtube1cIcon fill="currentColor" />}
```
Comment (TrySound) [inline apps/builder/app/builder/sidebar-left/help-popover.tsx:48]: I guess it will be in a lot of places

File: packages/icons/generate.ts [hunk 2]
```diff
@@ -56,15 +56,15 @@ const generateComponentExport = (options: GenerateOptions) => {
...
+      fill: "{fill}",
       "{...props}": null,
       ref: "{forwardedRef}",
     },
     plugins: sharedPlugins,
   });
   return `
 export const ${options.exportName}: IconComponent = forwardRef(
-  ({ color = "currentColor", size = 16, ...props }, forwardedRef) => {
+  ({ color = "currentColor", fill = "none", size = 16, ...props }, forwardedRef) => {
```
Comment (TrySound) [inline packages/icons/generate.ts:67]: What is color then?

Comment (kof) [reply packages/icons/generate.ts:67]: well, idk if worth renaming to strokeColor

File: apps/builder/app/builder/sidebar-left/help-popover.tsx (hunk 1 above)
Comment (kof) [reply apps/builder/app/builder/sidebar-left/help-popover.tsx:48]: no, only in those filled icons like the brand icons, 999.9% are stroke icons

File: packages/icons/generate.ts (hunk 2 above)
Comment (kof) [reply packages/icons/generate.ts:67]: actually we don't set it much, so its easy to rename

**Review by TrySound — APPROVED** [review]

**Pull request merged**